db/db_impl.cc

```diff
@@ -2064,7 +2064,8 @@ ColumnFamilyHandle* DBImpl::GetColumnFamilyHandleUnlocked(
     return nullptr;
   }
 
-  return cf_memtables->GetColumnFamilyHandle();
+  return std::unique_ptr<ColumnFamilyHandleImpl>(
```

Comment:
Don't hack this, if you need to make sure the lifetime of the handle, you can keep it when opening or creating the column family in TitanDB.

Reply:
This implementation will not stop user dropping `ColumnFamilyHandle` and we should never do that. It will just make sure that we won't hold a dangle pointer to `ColumnFamilyData`.